app/Http/Controllers/ProjectController.php

```diff
@@ -40,10 +40,11 @@ public function store(Request $request) {
 			'longDescription' => ['present'],
 			'shortDescription' => ['present'],
 			'projectImage' => ['present'],
-			'category' => [
-				'present',
-				// verify category to be valid (exists as a category tag record)
-				Rule::in(Tag::where('type', 2)->pluck('id'))
+			'category' => 'present|array|min:1',
```

Comment:
lo podríamos renombrar a `tag`? siento que es más descriptivo que category porque no necesariamente son puras categories 😨

Reply:
agree! aunq internally podría estar confuso bc los "skills" también se guardan como tags en la bd.
pero podriamos ponerle `lable` y evitar confusión in the long run ?